mentoring-wg/gsoc-org-admin-guide.md

```diff
@@ -0,0 +1,163 @@
+    * We are no more collecting ideas from mentors.
+* Inform member mentors (mentors who added an idea) about these:
+    * They need to review proposals from contributors before they submit the final proposal.
+* As we are not collecting ideas anymore, finalize getting mentors:
+    * Added to GSoC platform
+    * Added to GSoC program for current year in GSoc platform
+
+
+### Rankings
+
+* Ask mentors to
+    * Review proposals and select contributors they want to mentor
+    * Provide feedback in the GSoC system around the matters listed in the [contributor+proposal selection and ranking criteria](#Contributorproposal-selection-and-ranking-criteria), such as community interaction, proposal quality, etc.
+    * Do not comment on chance of acceptance to a contributor
+* Run the ranking process
+
+### Accepted projects announcement
+
+* [Announce](#Announcements) that accepted contributors and projects are announced.
+* Add selected mentors to the mailing list created earlier
+* Inform mentors and contributors about:
+    * Community bonding
+    * Getting up to speed to begin working on projects
+
+### Coding period
+
+* Inform mentors and contributors about:
+    * Coding started
+    * Midterm evaluation date
+    * TODO: anything to inform around monitoring their progress? While GSoC admins cannot technically review the progress, we can ask biweekly/monthly reports from mentors
+
+### Midterm evaluations
+* Inform mentors and contributors about:
+    * Midterm evaluations and deadlines
+    * [Google's evaluation guide](https://google.github.io/gsocguides/mentor/evaluations)
+
+
+### Final evaluations
+* Inform mentors and contributors about:
+    * Midterm evaluations and deadlines
+    * [Google's evaluation guide](https://google.github.io/gsocguides/mentor/evaluations)
+
+
+### Results
+* Run a retrospective using processes such as:
+    * Feedback survey for contributors and mentors
+    * Analyze data
+    * Retrospective session for GSoC admins
+* Fill Google's feedback survey.
+* [Announce](#Announcements) the results.
+
+### Additional tasks
+
+* Mentor stipend: TODO
+* Maintain this guide
+
+## Project idea proposal review guidelines
+
+* Make sure the mentor is in the [project-maintainers.csv](https://github.com/cncf/foundation/blob/main/project-maintainers.csv) file for the related project. If not, ask the PR author to ask for a `lgtm` from the people on that list.
+* Check if the formatting is right.
+* Make sure the upstream ticket actually exists and is open and unassigned.
+* Check if the project size in hours is one of the values provided by Google.
+* Check if the mentors have names, Github usernames and emails listed.
+* Make sure that the description provides a good sense of context and motivation for the idea to attract contributors.
+* Also make sure the expected outcome is describing something valuable.
```

Comment:
"Describing something valuable" could also be a vague concept here, thought of differently across mentors and projects. I'm wondering if we can further qualify this as "Make sure the expected outcome has benefits for contributors and the project", with "benefits" here being learning, growth, new features/functionality, etc

Reply:
Updated to

> * Make sure the expected outcome has benefits for contributors (e.g. learning, growth) and the project (e.g. new features/functionality, bug fixes).